extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -1133,4 +1133,21 @@ class HiveCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
       }
     }
   }
+
+  test("Add/List Jars/Files/Archives command") {
+    withTempDir { path =>
+      withSingleCallEnabled {
+        Seq("JAR", "FILE", "ARCHIVE").foreach { cmd =>
+          doAs(admin, sql(s"ADD $cmd '$path'"))
+          interceptContains[AccessControlException](
+            doAs(someone, sql(s"ADD $cmd '$path'")))(
+            s"does not have [select] privilege on [[$path, $path/]]")
+          doAs(admin, sql(s"LIST $cmd '$path'"))
+          interceptContains[AccessControlException](
+            doAs(someone, sql(s"LIST $cmd '$path'")))(
+            s"does not have [select] privilege on [[$path, $path/]]")
```

Comment:
These are all wrong. If you use `LIST FILE`, you will get `path` returned, but if you specify `path` to  `LIST FILE`, errored?

Reply:
> These are all wrong. If you use `LIST FILE`, you will get `path` returned, but if you specify `path` to `LIST FILE`, errored?

Yea。。。forgot this...